### PR TITLE
Replaced some fwlinks in standard folder

### PR DESCRIPTION
**Pull request by pkulikov**
This time **standard/data/xml** folder.
Related to #3425 

Didn't manage to find *XML Developer Center*, so just removed the whole link.

### CODE REVIEW

File: docs/standard/data/xml/xslt-compiler-xsltc-exe.md
Comment (mairaw) [inline docs/standard/data/xml/xslt-compiler-xsltc-exe.md]: Not related to your change but if you don't mind fixing:
- remove 2008 from the sentence
- change the link to `[Visual Studio Downloads(https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs)` (it's a special campaign link for VS)

File: docs/standard/data/xml/xslt-transformations.md
Comment (mairaw) [inline docs/standard/data/xml/xslt-transformations.md]: Probably would be better to link to https://www.w3.org/TR/xslt/ and change the link text to "W3C XSL Transformations (XSLT) recommendation" without version to be future-proof

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @pkulikov. Changes look great! Left a couple of suggestions.

**Comment by pkulikov** [comment]
@mairaw thank you for feedback, I've addressed it and pushed one more commit where I've looked at `docs/standard/data/xml/xslt-transformations.md` file more carefully. Please check.

File: docs/standard/data/xml/xslt-transformations.md
Comment (mairaw) [inline docs/standard/data/xml/xslt-transformations.md]: @krwq, @pjanotti here and in other places of the docs we mention that XslCompiledTransform supports the XSLT 1.0 syntax. However, XSLT is now on version 3.0. Are we still only supporting version 1.0 or do we support other versions too?

File: docs/standard/data/xml/xslt-transformations.md
Comment (mairaw) [inline docs/standard/data/xml/xslt-transformations.md]: We should file an issue to review this area. We shouldn't have topics that cover classes that are obsolete for so long. That should probably be changed to use XslCompiledTransform instead.

**Review by mairaw — COMMENTED** [review]
Thanks for addressing my comments. I'd like to confirm the XLST version supported by XslCompiledTransform before merging this.

File: docs/standard/data/xml/xslt-transformations.md
Comment (pjanotti) [reply docs/standard/data/xml/xslt-transformations.md]: Only 1.0, see https://github.com/dotnet/corefx/issues/2295

File: docs/standard/data/xml/xslt-transformations.md
Comment (pjanotti) [reply docs/standard/data/xml/xslt-transformations.md]: Actually we un-obsoleting this type :smile: there are a few scenarios that one should use `XslTransform` see https://github.com/dotnet/corefx/issues/25874 and a brief description of why at  https://github.com/dotnet/corefx/issues/25874#issuecomment-351465098

Comment (mairaw) [reply docs/standard/data/xml/xslt-transformations.md]: Thanks for the heads-up. I see that I missed the tag on that issue. Let's add an issue for the required docs work then to un-obsolete that type. 😄

File: docs/standard/data/xml/xslt-transformations.md
Comment (mairaw) [inline docs/standard/data/xml/xslt-transformations.md]: Given @pjanotti prompt answer, I'll have to take it back my previous feedback on this. So let's switch back to https://www.w3.org/TR/xslt-10/ and add the Version 1.0 back here.

Let me know if you'd like me to do that @pkulikov given I asked you to change that before.

Comment (pkulikov) [reply docs/standard/data/xml/xslt-transformations.md]: @mairaw no problem: I'll change it back! Please create the issue for un-obsoleting another type and that PR can be merged at last :).

**Review by mairaw — APPROVED** [review]
Thank you for making the changes @pkulikov. We'll merge this once build looks good.

**Pull request merged**
